extensions/sql-database-projects/package.json

```diff
@@ -43,9 +43,9 @@
       {
         "title": "%sqlDatabaseProjects.Settings%",
         "properties": {
-          "sqlDatabaseProjects.netCoreSDKLocation": {
+          "sqlDatabaseProjects.dotnetSDK Location": {
```

Comment:
I'd be careful of changing this - it means that anyone who has this set is going to lose it. I'd suggest adding an upgrade path from the old name in addition to changing this.

Reply:
Would a reasonable upgrade path be something like keep the old setting and add the new setting this release, and set the new setting's value(if none yet) to the old setting's value, then remove the old setting next release?

Reply:
I'd leave the JSON as you have it and then just check if the old one has a value on startup - if it does then copy that over to the new one and clear the old one. Something like this I don't think we need to incrementally deprecate.

Reply:
Updated to copy over the old value to the new one, but I couldn't make it clear the old one because there's an error that it can't update the old setting to clear it since it isn't a registered configuration anymore.

old setting getting copied over to the new one:
![updateSetting](https://user-images.githubusercontent.com/31145923/143953526-2a4cbe23-f274-49dd-954e-26c2a73d84fe.gif)
.